src/Behat/Gherkin/Node/TableNode.php

```diff
@@ -73,12 +63,7 @@ public function __construct(array $table)
                 }
 
                 if (!is_scalar($string)) {
-                    throw new NodeException(sprintf(
-                        "Table cell at row '%s', col '%s' is expected to be scalar, got %s",
-                        $ridx,
-                        $column,
-                        gettype($string)
-                    ));
+                    throw new NodeException(sprintf("Table cell at row '%s', col '%s' is expected to be scalar, got %s", $ridx, $column, gettype($string)));
```

Comment:
Looks like we still have some things converted to (long) one-line exceptions?

Reply:
@acoulton the problem is that if I redo the whole process from scratch, I would lose the manual fixes. So things like the long exception lines have to be fixed manually (since the rule can only be enabled or disabled - not reversed).

In this particular case, I'll check all lines starting with "throw new ".

Reply:
I've fixed them now.